### ISSUE TITLE
originalFileName in AssetInfo

### ISSUE THREAD

**Issue by jakearchibald** [rollup/rollup#2968]
### Feature Use Case

I want to create a map of original asset names to their hashed output file (for server-side rendering). `AssetInfo` only provides the final file name.

### Feature Proposal

`originalFileName` would be enough. However I guess https://github.com/rollup/rollup/issues/2959 could complicated this if two assets with different names are deduped. Maybe it would need to be an array, `originalFileNames`.

**Comment by lukastaegert** [comment]
Makes sense, though I would prefer `name` or `assetName` (or maybe `names`) as there is no reason that the provided name corresponds to an actually existing file.
